x/ccv/provider/keeper/keeper.go

```diff
@@ -178,6 +178,44 @@ func (k Keeper) DeleteChainToChannel(ctx sdk.Context, chainID string) {
 	store.Delete(types.ChainToChannelKey(chainID))
 }
 
+// SetChainsInProposal sets consumer chainId in gov consumerAddition proposal in store
+// the consumer chainId is only added when the voting period for consumerAddition proposal
+// does not end.
+func (k Keeper) SetChainsInProposal(ctx sdk.Context, chainID string, proposalID uint64) {
```

Comment:
Add unit tests for keeper methods.

Reply:
Simple setter and getter tests would be enough. No mocks are needed to execute those.